rss2email/feed.py

```diff
@@ -803,15 +803,15 @@ def _process_entry_content(self, entry, content, subject):
             if self.use_css and self.css:
                 lines.extend([
                         '    <style type="text/css">',
-                        self.css,
+                        _saxutils.escape(self.css),
```

Comment:
Whoever reading this and doubting it's correct because self.css is a user-configurable and not feed-provided: I think this is correct because one can escape CSS inline styles and it still works… or so I think.